com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/JarSignatureVerifier.java

```diff
@@ -0,0 +1,49 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.Collection;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sap.cloud.lm.sl.cf.process.Constants;
+import com.sap.cloud.lm.sl.cf.process.util.verifier.Verifier;
+import com.sap.cloud.lm.sl.common.SLException;
+
+@Component
+public class JarSignatureVerifier {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JarSignatureVerifier.class);
+
+    @Inject
+    private Verifier verifier;
+
+    public void checkCertificates(URL url, List<X509Certificate> certificates, String certificateCN) {
+        verifier.verify(certificates, certificateCN, url);
+    }
+
+    public List<X509Certificate> readProvidedCertificates(String certificatesFilename) {
+        Collection<? extends Certificate> certificates = null;
+        try (InputStream symantecInputStream = getClass().getResourceAsStream(certificatesFilename)) {
```

Comment:
Rename `symantecInputStream` to `certificateInputStream`.

Reply:
Done